numpy/ma/core.py

```diff
@@ -7341,6 +7341,118 @@ def size(obj, axis=None):
 size.__doc__ = np.size.__doc__
 
 
+def diff(a, n=1, axis=-1, prepend=np._NoValue, append=np._NoValue):
+    """
+    Calculate the n-th discrete difference along the given axis.
+    The first difference is given by ``out[i] = a[i+1] - a[i]`` along
+    the given axis, higher differences are calculated by using `diff`
+    recursively.
+    Preserves the input mask.
+    Parameters
```

Comment:
```suggestion

    Parameters
```
Missing newlines.  Could also cut this short and refer to the normal `np.diff` docs.